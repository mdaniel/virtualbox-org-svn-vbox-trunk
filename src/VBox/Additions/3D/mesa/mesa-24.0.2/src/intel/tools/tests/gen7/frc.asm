frc(8)          g19<1>.xF       (abs)g1<0>.xF                   { align16 1Q };
frc.sat(8)      g116<1>F        g6<4>F                          { align16 1Q };
frc(8)          g3<1>F          g2<0,1,0>F                      { align1 1Q };
frc(16)         g3<1>F          g2<0,1,0>F                      { align1 1H };
