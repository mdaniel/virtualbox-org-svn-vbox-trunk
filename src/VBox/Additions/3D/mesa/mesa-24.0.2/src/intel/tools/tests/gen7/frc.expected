43 01 60 00 bd 03 61 22 20 20 00 00 00 00 00 00
43 01 60 80 bd 03 8f 2e c4 00 6e 00 00 00 00 00
43 00 60 00 bd 03 60 20 40 00 00 00 00 00 00 00
43 00 80 00 bd 03 60 20 40 00 00 00 00 00 00 00
